packages/shared/routes/dashboard/wallet/views/Send.svelte

```diff
@@ -34,6 +34,11 @@
     // This looks odd but sets a reactive dependency on amount, so when it changes the error will clear
     $: amount, (amountError = '')
 
+    const sendSubscription = sendParams.subscribe(s => {
+        selectedSendType = s.isInternal ? SEND_TYPE.INTERNAL : SEND_TYPE.EXTERNAL
```

Comment:
After choosing to delete an account and getting into the internal transfer view, the tab "Send Payment" can no longer be selected because this line is overriding the decision

Reply:
I will take another look

Reply:
I added a fix which updates the selectedSendType by setting the sendParam instead of the internal var on chosing a tab